alacritty.yml

```diff
@@ -321,6 +321,11 @@
 # Window opacity as a floating point number from `0.0` to `1.0`.
 # The value `0.0` is completely transparent and `1.0` is opaque.
 #background_opacity: 1.0
+#
+# Whether or not `background_opacity` applies to all backgrounds or only the
+# default background.  When set to `true` it will only apply to the default
+# background and keep other background colors opaque.
+#keep_background_colors_opaque: true
```

Comment:
I think the naming of this option isn't ideal, since it says little about what it does in its name and I'd advocate for moving this option to `window` instead of keeping it at the top level. Together with `background_opacity` probably.

Reply:
I am open to suggestions for a better name. My 'inspiration' for naming it that way came from iterm (not that that is a strong argument):
<img width="427" alt="image" src="https://user-images.githubusercontent.com/3343670/92203509-3a03c300-ee79-11ea-977e-2438bfa6fc5e.png">

Happy to move it to the window section, do you want me to move only the new option for now so that it is in the right place to start with or also `background_opacity`? If the latter what is the procedure to deprecate configuration options or is a breaking change fine?

Reply:
> My 'inspiration' for naming it that way came from iterm (not that that is a strong argument):

I mean that's definitely a solid argument, thanks for letting me know. I'm struggling to suggest a much better alternative though. My main problem is that the `keep_` prefix doesn't really fit in well with our configuration options. Maybe something more like `opaque_colored_backgrounds`?

If you have some other suggestions, please do let me know.

> Happy to move it to the window section, do you want me to move only the new option for now so that it is in the right place to start with or also background_opacity? If the latter what is the procedure to deprecate configuration options or is a breaking change fine?

I think I'd move both, just to have it all in place. Though I'm not entirely sure if this should be under `colors` or `window`. @kchibisov any thoughts?

Reply:
it should be under window, I'd say.

Reply:
>I mean that's definitely a solid argument, thanks for letting me know. I'm struggling to suggest a much better alternative though. My main problem is that the keep_ prefix doesn't really fit in well with our configuration options. Maybe something more like opaque_colored_backgrounds?

We can have something like `apply_opacity_to_background_colors: false` setting, but I don't really mind original option.

Reply:
That's a tad long.

Reply:
I've changed it your suggestion.